From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Simonas Leleiva <simonas.leleiva@meramo.co.uk>
Date: Fri, 18 Feb 2022 08:18:31 +0000
Subject: [PATCH] (hybris) device: sony: common: Workaround when qcrild starts too early

[hybris] device: sony: common: Workaround when qcrild starts too early. JB#56682

Signed-off-by: Simonas Leleiva <simonas.leleiva@jolla.com>
---
 rootdir/vendor/etc/init/qcrild.rc | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/rootdir/vendor/etc/init/qcrild.rc b/rootdir/vendor/etc/init/qcrild.rc
index 6683aed58dc255a6cb23f08415bbdb379073a4dc..e002c303a8a8998b8454935c5fcc7f35a23bc999 100644
--- a/rootdir/vendor/etc/init/qcrild.rc
+++ b/rootdir/vendor/etc/init/qcrild.rc
@@ -1,4 +1,5 @@
-service vendor.qcrild /odm/bin/hw/qcrild
+# Workaround when sometimes qcrild starts too early
+service vendor.qcrild /system/bin/sh -c "sleep 3; /odm/bin/hw/qcrild"
     class main
     user radio
     disabled
